scanner/src/main/kotlin/scanners/scanoss/ScanOss.kt

```diff
@@ -47,30 +56,79 @@ class ScanOss internal constructor(
             ScanOss(scannerName, scannerConfig, downloaderConfig)
     }
 
+    private val config = ScanOssConfig.create(scannerConfig).also {
+        log.info { "The $scannerName API URL is ${it.apiUrl}." }
+    }
+
+    private val service = ScanOssService.create(config.apiUrl)
+
+    companion object {
+        // The name of the multipart body being sent. Has no importance.
```

Comment:
Maybe: s/Has no importance./Can be arbitrary./

Or yet more compact: "An arbitrary name to use for the multipart body being sent."